[DOCS-1573] Document Azure pre-signed URL requesting username field (#1379)
wandb/docs: content/guides/hosting/data-security/presigned-urls.md

{{% /alert %}}

### Determine the user that requested a pre-signed URL

> When W&B returns a pre-signed URL, the response includes the requester's username:

Author: ```suggestion
When W&B returns a pre-signed URL, a query parameter in the URL contains the requester's username:
```

> | Storage provider   | Response field  |

Reviewer: It's not the response from W&B that has the `X-User` header, it's a query parameter of the signed URL that W&B returns. That signed URL then is used by the client to perform storage operations, which include the `X-User` header. That's the case for AWS and GCP. 

For Azure, as I understand, the client doesn't forward the query parameters to the storage requests as headers. Instead, the pre-signed URL used can be checked directly.

Something like this:
```suggestion
| Storage provider   | Signed URL Query Parameter |
```

> | Storage provider   | Response field  |

Author: ```suggestion
| Storage provider   | Signed URL query parameter  |
```

|--------------------|-----------------|

> | AWS bucket         | `X-User` header |

Reviewer: AWS S3 might be more correct terminology to match line 42?

> | AWS bucket         | `X-User` header |
> | GCP bucket         | `X-User` header |
> | Azure blob storage | `scid` query parameter |

Author: ```suggestion
| AWS S3  storage           | `X-User`  |
| Google Cloud storage   | `X-User` |
| Azure  blob storage       | `scid`      |
```
